api/backup.go

```diff
@@ -101,6 +114,9 @@ type Backup struct {
 	MigrationHistoryVersion string `jsonapi:"attr,migrationHistoryVersion"`
 	Path                    string `jsonapi:"attr,path"`
 	Comment                 string `jsonapi:"attr,comment"`
+	// Payload contains data such as backup position info which will not be created at first.
+	// Backup runner will fill in this field.
```

Comment:
Needs to comment under which occasion, Backup runner will fill in this field

Reply:
Done.